deploy/clusterrole.yaml

```diff
@@ -51,3 +51,13 @@ rules:
   - 'cronjobs'
   verbs:
   - 'get'
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
```

Comment:
Do we not `delete` the apiservice ? If so, we'll need to add the verb here.

Reply:
Yes, I missed this one, it's odd that it's able to delete it.

Reply:
That's what we do in this code path correct https://github.com/DataDog/datadog-operator/pull/137/files#diff-7554a7176dee880f102333a54563b582R255 ?